### PR TITLE
docs: Lg/cli 0.5.5 CLI logging options for dfx

### PR DESCRIPTION
**Pull request by lsgunnlsgunn**
Need more info / examples to make the descriptions useful

### CODE REVIEW

File: modules/developers-guide/pages/cli-reference.adoc [hunk 1]
```diff
@@ -108,6 +108,21 @@ For example, to see usage information for `+dfx build+`, you can run the followi
...
+If you don't specify any logging level, CRITICAL, ERROR, WARNING, and INFO messages are logged by default.
+Specifying one verbose flag (`+-v+`) increases the log level to include DEBUG messages. 
+Specifying two verbose flags (`+-vv+`)increases the logging level to include both DEBUG and TRACE messages.
+
+Adding a `+--quiet+` flag decreases the logging level.
+For example, to remove all messages, you can run a command similar the following:
+
+[source,bash]
+----
+dfx build -qqqq
```
Comment (hansl) [inline modules/developers-guide/pages/cli-reference.adoc:121]: `-qqqq` should be after `dfx` and before anything else, e.g. `dfx -qqqq build`.

Also, not all commands support it yet.

Comment (lsgunnlsgunn) [reply modules/developers-guide/pages/cli-reference.adoc:121]: which commands support the verbose and quiet flags?

Comment (lsgunnlsgunn) [reply modules/developers-guide/pages/cli-reference.adoc:121]: Also, the syntax conventions seem a bit inconsistent. For some subcommand/flag/option combinations, this convention seems to be required:
dfx [subcommand] [flag] [option]
but for other subcommand/flag/option combinations, this convention does not appear to be supported. Or is there a nuance that I'm missing here?
If, in all cases, it doesn't matter about the order, then the syntax summary is fine. And maybe it doesn't even matter if summary doesn't always apply if the error message always specifies the correct order for any given command. But it is sort of weird to make someone guess the correct order to make a specific command work then not be sure if a different command requires the same order or a different order.

Comment (hansl) [reply modules/developers-guide/pages/cli-reference.adoc:121]: `dfx` itself supports it, so you need to pass it before the subcommand. So it's really `dfx [dfx-options] [subcommand] [subcommand-options]` if that makes sense. This is the same concept as the `--client` argument for the `canister` subcommand.

`canister install` and `new` use it. We're slowly moving everything to use this in future releases.

**Comment by lsgunnlsgunn** [comment]
Could still use some examples for using the log mode and logging-file options.

**Pull request merged**